source/getting-started/faq.rst

```diff
@@ -459,7 +464,7 @@ After you have created a key pair and imported it in Chameleon, you can connect
 .. _faq-ssh-keypairs-windows:
 
 For Windows
-___________
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
What does this change do? And should the line be smaller?

Reply:
So "Create your own SSH key pairs on Linux/macOS" used to be header level 1. I created a new header level 1 called "SSH Issues" and put "Create your own SSH key pairs on Linux/macOS" under that as header level 2. Then I have to make "For Windows" header level 3 (originally header level 2). Does it make sense?